### DISCUSSION TITLE
Equivalent API to kubectl wait?

### DISCUSSION THREAD

**Discussion by smithago** [kubernetes-client/csharp#874] [Q&A]
We have CRD dicoms. I use `CreateNamespacedCustomObjectAsync` to create a new instance.

What is the equivalent of something like
```cli
 kubectl wait --for=condition=Ready dicoms smitest03 --timeout=60s
```

**Comment by brendandburns** [comment]
You can either use [watch](https://github.com/kubernetes-client/csharp/blob/master/examples/watch/Program.cs) or poll on `ReadNamespacedCustomObject`

**Comment by smithago** [comment]
Thanks, I did a poll
